src/site/content/en/reliable/http-cache/index.md

```diff
@@ -1,52 +1,77 @@
-title: 'The HTTP cache: your first line of defense'
+title: 'The HTTP Cache: Avoiding unnecessary network requests with `Cache-Control` and `ETag`'
 authors:
   - jeffposnick
+  - ilyagrigorik
 date: 2018-11-05
-description: |
-  The browser's HTTP cache is your first line of defense. It's not necessarily
-  the most powerful or flexible approach, and you have limited control over the
-  lifetime of cached responses. But there are several rules of thumb that give
-  you a sensible caching implementation without much work, so you should always
-  try to follow them.
+updated: 2020-04-17
+description: >
+  How can you avoid unnecessary network requests? The browser's HTTP Cache is your
+  first line of defense. It's not necessarily the most powerful or flexible
+  approach, and you have limited control over the lifetime of cached responses,
+  but it's effective, it's supported in all browsers, and it doesn't require much
+  work.
 codelabs:
   - codelab-http-cache
 ---
 
-How can you avoid unnecessary network requests?
+Fetching resources over the network is both slow and expensive: 
 
-The browser's HTTP cache is your first line of defense. It's not necessarily the
-most powerful or flexible approach, and you have limited control over the
-lifetime of cached responses. But there are several rules of thumb that give you
-a sensible caching implementation without much work, so you should always try to
-follow them.
+* Large responses require many roundtrips between the browser and the server.
+* Your page won't load until all of its [critical resources][crp] have downloaded completely.
+* If a person is accessing your site with a limited mobile data plan, every unnecessary 
+  network request is a waste of their money.
 
-The HTTP cache's behavior is controlled by a combination of
-[request](https://developer.mozilla.org/en-US/docs/Glossary/Request_header) and
-[response](https://developer.mozilla.org/en-US/docs/Glossary/Response_header)
-headers. In an ideal scenario, you'll have control over both the code for your
+How can you avoid unnecessary network requests? The browser's HTTP Cache is your
+first line of defense. It's not necessarily the most powerful or flexible
+approach, and you have limited control over the lifetime of cached responses,
+but it's effective, it's supported in all browsers, and it doesn't require much
+work.
+
+This guide shows you the basics of an effective HTTP caching implementation.
+
+## Browser compatibility {: #browser-compatibility }
+
+There isn't actually a single API called the HTTP Cache. It's the general name
+for a collection of web platform APIs. Those APIs are supported in all browsers:
+
+* [`Cache-Control`](https://developer.mozilla.org/docs/Web/HTTP/Headers/Cache-Control#Browser_compatibility)
+* [`ETag`](https://developer.mozilla.org/docs/Web/HTTP/Headers/ETag#Browser_compatibility)
+* [`Last-Modified`](https://developer.mozilla.org/docs/Web/HTTP/Headers/Last-Modified#Browser_compatibility)
```

Comment:
It's a little weird that `Cache-Control` and `ETag` are specifically called out in the article's title, but there are actually three things listed here.

Reply:
If you're OK with the new title then this should be resolved now.